tests/ftests/ftests-nocontainer.sh

```diff
@@ -21,10 +37,10 @@ fi
 	-n Libcg"$RANDOM"
 RET1=$?
 
-pushd ../../src || exit $AUTOMAKE_HARD_ERROR
+pushd ../../src || cleanup $AUTOMAKE_HARD_ERROR
 PATH="$PATH:$(pwd)"
 export PATH
-popd || exit $AUTOMAKE_HARD_ERROR
+popd || cleanp $AUTOMAKE_HARD_ERROR
```

Comment:
typo - should be cleanup :)

Reply:
Sorry, it should have been `cleaup`. Will fix it up in the next iteration.